DevelopersGuide.adoc

```diff
@@ -738,6 +738,8 @@ of Shadow-cljs for more information.
 
 It also has hot-code (and CSS) reload built in, so there is no need for any additional tools!
 
+Build your app now by selecting "main" under the "Builds" menu and clicking "start watch".
```

Comment:
This seems repetitive to me from line 732

Reply:
> You can then use the "Builds" menu to turn on/off different client builds and see the progress live as it happens!

Instructions telling me I can turn something on _and off_ strongly suggest optional to me, and so I didn't see the "TODO" when the app loaded. I was hoping to help those that follow in my footsteps, but maybe I'm an outlier.